detd/devices/device.py

```diff
@@ -125,7 +118,8 @@ def supports_schedule(self, schedule):
         '''
 
         raise NotImplementedError("The handler class for the device must implement this function")
-
-
-    def supports_qbv(self):
-        return Capability.Qbv in self.capabilities
+    
+    def device_hints(self):
+        '''Returns device supported Hints.
```

Comment:
I understand that this method returns the default hints for a given device. If that is the case, we may include a default value for any device, where we provide:
* TxSelection: EST
* TxSelectionOffload: false
* DataPath: AF_PACKET
* Preemption: false
* Launch Time Control: false

In fact, this selection can be made by a generic algorithm defined in this function:

* If device supports Qbv in HW: set selection and offload true accordingly
* Else: like above but offload false

* Set datapath as AF_PACKET

* Set LTC and Preemption as false

This would force the use of EST with AF_PACKET, with or without HW offload depending on the device capabilities.

But maybe it is easier to understand if we provide a fixed default in this plugin, and have the full hints calculation per device plugin. This will probably make the code easier to follow.

In addition, perhaps then we can rename the method and description to emphasize its nature. E.g. a device may support multiple hints, but will have a predefined default hints.